### DISCUSSION TITLE
NIST CVSS vs OpenSSL's security assessment divergence on CVE-2023-0464

### DISCUSSION THREAD

**Discussion by yelled1** [openssl/openssl#22124] [Q&A]
Can someone from OpenSSL help me understand the divergence?

https://www.openssl.org/news/secadv/20230322.txt says:
OpenSSL Security Advisory [22nd March 2023]
Excessive Resource Usage Verifying X.509 Policy Constraints (CVE-2023-0464)
Severity: **Low**
Due to the **low severity** of this issue we are not issuing new releases of OpenSSL at this time.

However, in https://nvd.nist.gov/vuln/detail/CVE-2023-0464 CVSS 3.x Severity and Metrics says:
NIST CVSS scoreNIST: NVDBase **Score: 7.5 HIGH**

BTW: I have asked NIST to explain as well by sending an email. Will add NIST's response once & if I get one, as this had caused a painful exception request since  it's discovery & another exception review is coming up in a week or so.

**Comment by t8m** [comment]
The CVSS score does not account for the likelihood of an application being vulnerable. The policy checking is very rarely enabled by applications, so it is very unlikely that an application is affected. 
Also we never regard denial of service on the client side to be High severity. To impact a TLS server application, which is the most common use-case that could be potentially impacted, the server needs to enable client certificate authentication along with the policy check making the likelihood even lower.

**Comment by levitte** [comment]
NVD performs their own CVSS calculation.  This is somewhat clarified in their FAQ https://nvd.nist.gov/general/FAQ-Sections/CVE-FAQs : 

> Q. Why don’t scores provided by NVD match scores provided by vendors or other parties?
> A. The NVD assesses CVEs using publicly available information at the time of analysis. To ensure that the vector strings in the NVD reflect publicly available information and abide by CVSS specification guidelines, CVSS vector strings provided by third parties are not copied outright. This can lead to differences in CVSS vector strings between different parties. Usually NVD and vendor/third party vector strings differ due to information being overly vague or unavailable at the time of analysis. If you believe that a CVSS vector string should be revised, please contact the NVD using the [nvd.nist.gov/info/contact-form](https://nvd.nist.gov/info/contact-form) and provide publicly available information that corroborates any claims.

**Comment by yelled1** [comment]
@t8m @levitte Thank you for the heads up.
Also just filed a CVE Rescore Request on it's website.
